index.html

```diff
@@ -1,39 +1,348 @@
+              dummy text.
+            </p>
+            <div>
+              <ul class="languages">
+                <li><img src="./images/css.png" alt="" /></li>
+                <li><img src="./images/html.png" alt="" /></li>
+                <li><img src="./images/bootstrap.png" alt="" /></li>
+                <li><img src="./images/ruby.png" alt="" /></li>
+              </ul>
             </div>
-            <div><img src="./Images/Group.png" alt="Background Image"></header></div>
-        </header>
-       
-       
+            <button>See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <!-- projects -->
+      <div class="projects-grid">
+        <div class="project">
+          <div class="project-header">
+            <h1>Professional Art Printing Data</h1>
+          </div>
+          <div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div>
+            <ul class="projectLanguages">
+              <li><img src="./images/Tag (4).png" alt="" /></li>
+              <li><img src="./images/Tag (5).png" alt="" /></li>
+              <li><img src="./images/Tag (6).png" alt="" /></li>
+            </ul>
+          </div>
+          <div>
+            <button>See Project</button>
+          </div>
+        </div>
+
+        <div class="project">
+          <div class="project-header">
+            <h1>Professional Art Printing Data</h1>
+          </div>
+          <div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div>
+            <ul class="projectLanguages">
+              <li><img src="./images/Tag (4).png" alt="" /></li>
+              <li><img src="./images/Tag (5).png" alt="" /></li>
+              <li><img src="./images/Tag (6).png" alt="" /></li>
+            </ul>
+          </div>
+          <div>
+            <button>See Project</button>
+          </div>
+        </div>
+
+        <div class="project">
+          <div class="project-header">
+            <h1>Professional Art Printing Data</h1>
+          </div>
+          <div>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard
+            </p>
+          </div>
+          <div>
+            <ul class="projectLanguages">
+              <li><img src="./images/Tag (4).png" alt="" /></li>
+              <li><img src="./images/Tag (5).png" alt="" /></li>
+              <li><img src="./images/Tag (6).png" alt="" /></li>
+            </ul>
+          </div>
+          <div>
+            <button>See Project</button>
+          </div>
+        </div>
```

Comment:
- [ ] Good job implementing this `project section` :clap: , However there is something you need to change to meet the requirements. :100: . Note that the `see Project` button is not visible initially, It's when the mouse is over that you can see it. Kindly change that :smile: 

Your Design

![Screenshot from 2022-12-10 18-09-58 (1)](https://user-images.githubusercontent.com/80458199/206865623-3b4fbdb8-2c6c-4f45-ba11-b5d5001bc257.png)


Figma Design
![Screenshot from 2022-12-10 18-09-24 (1)](https://user-images.githubusercontent.com/80458199/206865586-cf1bc4da-3c99-4889-b9a8-ed0a4602226e.png)